client/src/components/PersonCard/personcard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import "./personcard.css";
+import mom from "../Assets/Images/Mom.png"
+
+// Function based React Component
+const PersonCard = (props) => {
+
+  // Default Class to apply to Component
+  let classList = `PersonCard`;
```

Comment:
Can you put this class in the `className` attribute, rather than create an extra variable for it?